crates/bevy_window/src/window.rs

```diff
@@ -559,10 +560,27 @@ pub struct WindowDescriptor {
     pub canvas: Option<String>,
 }
 
+static DEFAULT_WINDOW_TITLE: Lazy<String> = Lazy::new(|| {
+    std::env::current_exe()
+        .ok()
+        .and_then(|it| Some(format!("{} - bevy", it.file_stem()?.to_string_lossy())))
+        .unwrap_or_else(|| "bevy".to_string())
+});
+
+impl WindowDescriptor {
+    pub fn title(&self) -> &str {
```

Comment:
Given that title is a public field, i think adding an accessor that doesn't match that field is a bit confusing. Maybe we should change this to `title_or_default()` or something in that vein?